Cargo.lock

```diff
@@ -636,9 +636,9 @@ checksum = "d468802bab17cbc0cc575e9b053f41e72aa36bfa6b7f55e3529ffa43161b97fa"
 
 [[package]]
 name = "aws-config"
-version = "0.56.1"
+version = "0.56.0"
```

Comment:
why is it downgrading aws related dependencies?

Reply:
I’m not sure, I only ran `cargo update -p webpki -p rustls-webpki`

Reply:
fixed in fde03d00772be3a79a3f45cad8ac91c01c9c8365